installer/manifests/keptn/templates/shipyard-controller.yaml

```diff
@@ -82,7 +82,7 @@ spec:
             - name: LOG_LEVEL
               value: {{ .Values.logLevel | default "info" }}
             - name: AUTOMATIC_PROVISIONING_URL
-              value: {{ (.Values.features).automaticProvisioningURL | default "" | quote }}
+              value: {{ (.Values.features).automaticProvisioning.serviceURL | default "" | quote }}
```

Comment:
pls remove brackets

Reply:
I was told those are needed with new var to avoid the helm update to fail

Reply:
I am adding them everywhere 😄

Reply:
ok, also an option :+1: